cmd/vm.go

```diff
@@ -88,4 +58,10 @@ func init() {
 		_ = fmt.Errorf("kvm option not supplied")
 	}
 
+	// options for vm generic commands related to HTTP Request
+	vmCmd.Flags().StringVar(&HttpIP, "http-ip", "http://127.0.0.1", "IP of non-k8s control plane")
+	vmCmd.Flags().StringVar(&HttpPort, "http-port", "8080", "IP and port of http request")
+
+	// Create http address
+	HttpAddress = HttpIP + ":" + HttpPort
```

Comment:
When HttpAddress is instantiated the values of IP and Port are nil since the flags are not parsed. You need to declare this under `Run` method. This is the reason why you are not getting values as needed.